uwrt_mapviz/README.md

```diff
@@ -0,0 +1,73 @@
+# uwrt_mapviz
+
+## Installation
+
+## Installing mapviz
+
+### Install from Ros-binaries
+```
+$ sudo apt-get install ros-kinetic-mapviz ros-kinetic-mapviz-plugins ros-kinetic-tile-map
```

Comment:
These lines shouldn't be required. The only thing you should typically need to do to install a ros workspace's dependencies should be running `rosdep install --from-paths src --ignore-src -r -y`. It automatically finds all required dependencies based off your package.xml's